### DISCUSSION TITLE
Brew auto create symlinks from /usr/local/bin to /usr/local/Cellar. I remember I had to do it manually before.

### DISCUSSION THREAD

**Discussion by YunshuaiPeng** [Homebrew/discussions#1523] [Everyday usage]
So, Is this a new feature?

**Comment by carlocab** [comment]
I don't think so. I can't remember a time when `brew` didn't create symlinks like that.

**Reply by tjluoma** [reply]
`brew` always creates links from _Cellar_ to _bin_ unless a specific formula tells it not to, usually because it has the potential of confusing macOS.

This has been the standard behavior for as long as I can remember. Otherwise, you'd have to add every sub-folder in _Cellar_ to your `$PATH`.

**Reply by YunshuaiPeng** [reply]
Thanks. It must be I didn't notice.
